draft-ietf-oauth-attestation-based-client-auth.md

```diff
@@ -400,6 +406,66 @@ response_type=code&state=af0ifjsldkj&client_id=s6BhdRkqt3
 &code_challenge_method=S256&scope=account-information
 ~~~
 
+## Usage of Client Instance Key as DPoP Key {#combined-dpop}
+
+This section defines an OPTIONAL optimization that allows a single Proof of Possession (PoP) JWT to satisfy both (a) the Client Attestation PoP defined in this specification and (b) the DPoP proof defined in {{RFC9449}}. In this "combined mode" the Client Instance Key and the DPoP Key are the same asymmetric key pair, and a request using the mechanism carries only one PoP, the DPoP proof, instead of two separate PoP JWTs (the DPoP proof and Client Attestation PoP).
+
+### Validating HTTP requests
+
+In order to detect whether this combined mode is being used, the following conditions MUST be met:
+1. There is precisely one OAuth-Client-Attestation HTTP request header field, where its value is a single well-formed JWT conforming to the syntax outlined in [](#client-attestation-jwt).
+2. There is precisely one DPoP HTTP request header field, its value is conforming to the syntax outlined in {{RFC9449}}.
+3. There is no OAuth-Client-Attestation-PoP HTTP request header field present.
+
+Provided the above conditions are meet the following additional rules apply:
+1. The public key located in the DPoP proof MUST match the public key located in the `cnf` claim of the Client Attestation JWT.
```

Comment:
Should we be more specific around how this matching should occur, e.g if both JWK use thumbprint etc?